extension/js/common/api/account-servers/external-service.ts

```diff
@@ -29,7 +29,7 @@ export namespace FesRes {
   };
   export type ServiceInfo = { vendor: string; service: string; orgId: string; version: string; apiVersion: string };
   export type ClientConfiguration = { clientConfiguration: ClientConfigurationJson };
-  export type HelpFeedback = { sent: boolean };
+  export type HelpFeedback = {}; // eslint-disable-line @typescript-eslint/ban-types
```

Comment:
As we expect just empty response, there is no need for separate `HelpFeedback` type - it can be removed. And then `helpFeedback` method should be transformed to:
```ts
public helpFeedback = async (email: string, message: string): Promise<void> => {
    return await this.request(`/api/${this.apiVersion}/account/feedback`, { fmt: 'JSON', data: { email, message } });
};
```

Reply:
Thank you, Roma. It has now been updated.